src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1776,4 +1789,75 @@ class LeVADocumentUseCase extends DocGenUseCase {
         return theString?.replaceAll('-', '&#x2011;')
     }
 
+    @Override
+    String invokeCreateDocMethod(String methodName, Object [] args) {
+        Map repo = null
+        Map data = null
+        if (args.length > 0) {
+            repo = args[0]
+            if (args.length > 1) {
+                data = args[1]
+            }
+        }
+
+        switch(methodName) {
+            case "createCFTP":
```

Comment:
Without a clear proof that calling the createXXX methods from NonCPS code is causing problems, I don't see this change justified. The warning has been known to be there for a long time, but we never found problems in this concrete case, yet. My recommendation is to remove this change from the PR.

Reply:
@jafarre-bi do you see anything that we need to take from this PR? If not, we might as well close.